### PR TITLE
chore: head branch of team shake_your_botpress

### PR DESCRIPTION
**Pull request by franklevasseur**
Head branch of team shake_your_botpress

Members of the team are: 
- @franklevasseur 
- @samuelmasse 
- @daukadolt 
- @crochcunill

Bug fixes:

- fix(nlu): prevent creating an entity with same name as a system (**1pts**)
- fix(nlu): allow copy paste utterances (**2pts**)
- fix(studio): added translations in node and flow inspector (**8pts**)

**DO NOT SQUASH THIS BRANCH**

### CODE REVIEW

File: packages/studio-ui/src/web/views/Nlu/intents/utterances-state-utils.ts [hunk 1]
```diff
@@ -29,29 +30,48 @@ const slotNode = (slot: ParsedSlot, uttIdx: number): TextJSON => ({
...
+
   return _.chain(parsedSlots)
     .flatMap((pslot, i, all) => {
-      const from = _.get(all, `${i - 1}.cleanPosition.end`, 0)
+      const previousSlot = all[i - 1]
+      const from = previousSlot?.cleanPosition.start ?? 0
       const to = pslot.cleanPosition.start
-      return [textNode(utterance, from, to), slotNode(pslot, idx)]
+
+      const slotExists = allSlots.map(s => s.name).includes(pslot.name)
```
Comment (laurentlp) [inline packages/studio-ui/src/web/views/Nlu/intents/utterances-state-utils.ts:54]:
```suggestion
      const slotExists = allSlots.some(s => s.name === pslot.name)
```

File: packages/studio-be/src/studio/nlu/entities-service.ts [hunk 2]
```diff
@@ -22,8 +22,17 @@ const SYSTEM_ENTITIES = [
   'volume'
 ]
 
+const getEntityId = (entityName: string) =>
+  entityName
+    .trim()
+    .toLowerCase()
+    .replace(/[\t\s]/g, '-')
```
[the remark this answers is not in the document]
Comment (franklevasseur) [reply packages/studio-be/src/studio/nlu/entities-service.ts:29]: I can't really import it from `packages/studio-ui/src/web/views/Nlu/entities/entity-id.ts` as this code is in the `studio-be` package

Is there such a thing as `common` in the studio ? 

I guess I could replace it with simply `.toLowerCase()`

[the remark this answers is not in the document]
Comment (laurentlp) [reply packages/studio-be/src/studio/nlu/entities-service.ts:29]: Right! I think simply adding a lower case would make more sense. You can also leave it like that as it is not really complex. I'll let you decide!

[the remark this answers is not in the document]
Comment (franklevasseur) [reply packages/studio-be/src/studio/nlu/entities-service.ts:29]: I moved it to common

**Review by laurentlp — APPROVED** [review]
LGTM! Good job!

**Comment by franklevasseur** [comment]
I'd love to merge but I don't have enough rights on the repo to enable Merge commits. I don't believe head branches should be squashed.

@allardy @EFF @spg, can anyone merge this without squashing?

**Pull request merged**